docker/golang-base/Dockerfile

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2018
+# Intel
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+FROM golang:1.11.2-alpine3.7
+
+LABEL license='SPDX-License-Identifier: Apache-2.0' \
+      copyright='Copyright (c) 2018: Intel'
+
+RUN echo http://nl.alpinelinux.org/alpine/v3.7/main > /etc/apk/repositories; \
+    echo http://nl.alpinelinux.org/alpine/v3.7/community >> /etc/apk/repositories
+
+RUN apk update && \
+  apk add --no-cache linux-headers make gcc musl-dev git && \
+  apk add --no-cache --virtual=.build-dependencies &&\
```

Comment:
This code looks vestigial, what is it actually doing?

```
  apk add --no-cache --virtual=.build-dependencies &&\
  ln -s locale.h /usr/include/xlocale.h && \
  rm /usr/include/xlocale.h && \
  apk del .build-dependencies
```

Reply:
Agreed, this can be removed.